### ISSUE TITLE
[Feature Request] Prevent command messages

### ISSUE THREAD

**Issue by Epb7304** [lorddusk/HQM#690]
TL/DR:
Change the way commands are executed on quest complete to act like they are run through command blocks or add a separate gamerule to prevent seeing the output in chat.

The issue to be solved:
Currently, when executing commands on quest complete (an amazing feature btw) it can spam the chat with the commands that it is running
![image](https://github.com/lorddusk/HQM/assets/54673449/39ad9c02-12de-4c79-ab18-136d65dfdc05)

This can be somewhat mitigated by running `/gamerule sendCommandFeedback false` however that prevents all feedback, even some that would want. You might think that you can simply run `/gamerule sendCommandFeedback true` after each command set. However that leads to a line in chat whenever a quest is completed, which while that can be dealt with it is still inconvenient.

How:
as I see it there are 2 options to fix this. Option 1 would be to change how the commands are dealt with by the game, instead of the commands being run by a sudo-player (Mod-HQM); the game could be tricked into thinking they are command block commands. Then the gamerule `/gamerule commandBlockOutput false` would work on it, and since almost nobody wants to see command block output it would adequately fix the issue. Option 2 is to add a gamerule that specifically disables the output from quests, something along the lines of `/gamerule HQMCommandOutput` this could also be done in the config.

Why:
this change would prevent the chat from being clogged up with command outputs while not hindering what the player can do with commands by themselves.
